beacon-chain/db/deposits.go

```diff
@@ -82,6 +83,27 @@ func (db *BeaconDB) AllDeposits(ctx context.Context, beforeBlk *big.Int) []*pb.D
 	return deposits
 }
 
+// DepositsContainersTillBlock returns the deposit containers sorted by block height till block height.
+func (db *BeaconDB) DepositsContainersTillBlock(ctx context.Context, beforeBlk *big.Int) []*DepositContainer {
+	ctx, span := trace.StartSpan(ctx, "BeaconDB.AllDepositsWithTrieRoots")
+	defer span.End()
+	db.depositsLock.RLock()
+	defer db.depositsLock.RUnlock()
+
+	var deposits []*DepositContainer
+	for _, ctnr := range db.deposits {
+		if beforeBlk == nil || beforeBlk.Cmp(ctnr.block) > -1 {
+			deposits = append(deposits, ctnr)
+		}
+	}
+	// Sort the deposits by Merkle index.
```

Comment:
can be more explicit here ascending order or descending order?

Reply:
changed to
// Sort the deposits by Merkle index in an ascending order.